src/background/Wallet/GlobalPreferences.ts

```diff
@@ -26,6 +26,7 @@ export interface State {
   providerInjection?: ProviderInjection;
   autoLockTimeout?: number | 'none';
   walletNameFlags?: Record<string, WalletNameFlag[] | undefined>;
+  currency?: string;
```

Comment:
I decided to put currency into global settings, so we are able to reach it anywhere in background without passing the Wallet into props